contrib/install_db4.sh

```diff
@@ -31,7 +31,11 @@ sha256_check() {
   if check_exists sha256sum; then
     echo "${1}  ${2}" | sha256sum -c
   elif check_exists sha256; then
-    echo "${1}  ${2}" | sha256 -c
+    if [ "$(uname)" = "FreeBSD" ]; then
```

Comment:
I believe we work around this in depends by writing to a temp file rather than piping, as all tools manage to deal with that. I think this is fine here, though.

Reply:
Indeed! I also thought about rewriting the whole function - to hash the file, grab the output, then compare the hash ourselves. Unfortunately the output format is also different per tool, so it wouldn't get fully clean code either.
But yes - we could still do that later, at least this is a minor diff that obviously won't affect any other OS.